### ISSUE TITLE
Ability to have the carousel start over when reaching the end

### ISSUE THREAD

**Issue by denizdogan** [reinvanoyen/tnt-carousel#2]
I've set up my carousel like this (just one at this time):

``` javascript
// set up carousels
$('.carousel').each((_idx, c) => {
  new Carousel(c, {
    autoplay: true,
    autoplayInterval: 1000,
    arrowButtons: false
  })
})
```

I expected the default settings to automatically "restart" at the beginning, but apparently it just ends? Can this functionality be added or is there some setting I've missed?

**Comment by reinvanoyen** [comment]
Thanks for the suggestion. It indeed appears that functionality isn't there yet. I'll add it ASAP.

Later it might be interesting to add continuous/forever sliding.

**Comment by reinvanoyen** [comment]
This is now standard behavior. It is available in v1.2.9 on npm.
